### ISSUE TITLE
[Map Editor] "Flat" Terrain Group has deprecated Terrains

### ISSUE THREAD

**Issue by knyghtmare** [wesnoth/wesnoth#7133]
![image](https://user-images.githubusercontent.com/5283677/202120481-22882931-ff73-421b-b18c-9a2bbcc5006a.png)

Where are the new replacements for them? They should be shown in the editor terrain select UI.

## Build:
Latest Commit
Windows 10 Msys2
Github

## Expectation:
No deprecated terrain being selectable on the Terrain Selection sidebar.

**Comment by Wedge009** [comment]
Um, I just checked ced504a9 and don't see any deprecation markers on the terrains you show.

**Comment by knyghtmare** [comment]
So, I think running wmllint (and forgetting to tick "skip core dir" kind of flips the deprecated codes to the current ones). I think wmllint has to adjusted to skip some files or the files need to have the no-convert to pass those

**Comment by Wedge009** [comment]
I have limited experience with `wmllint`, I just ran it with what I assume are the default parameters via `make -C data/tools lint`. This runs it with `--dryrun` against core, multiplayer, themes, and campaigns directories, the 'dry run' being the key, I suppose. Are you saying your working copy was 'dirty' after running `wmllint` and that was what caused this issue?

**Comment by knyghtmare** [comment]
> I have limited experience with `wmllint`, I just ran it with what I assume are the default parameters via `make -C data/tools lint`. This runs it with `--dryrun` against core, multiplayer, themes, and campaigns directories, the 'dry run' being the key, I suppose. Are you saying your working copy was 'dirty' after running `wmllint` and that was what caused this issue?

yes. Since I forgetting to tick "skip core" when running it on add-ons

**Comment by Wedge009** [comment]
That being the case, this can be closed as invalid then?

**Comment by knyghtmare** [comment]
Maybe add `#wmllint noconvert` beside the affected conversion cases? Because apart from terrain codes, some credits names are being modified as well.

**Comment by stevecotton** [comment]
> some credits names are being modified as well

That's worrying, AFAICS it's running a global search-and-replace for the string `Irs` with no context-checking.

**Comment by knyghtmare** [comment]
> That's worrying, AFAICS it's running a global search-and-replace for the string Irs with no context-checking.

It's been around far longer. Since my name keeps getting corrected to Dune Paragon.

**Comment by cooljeanius** [comment]
> > That's worrying, AFAICS it's running a global search-and-replace for the string Irs with no context-checking.
> 
> It's been around far longer. Since my name keeps getting corrected to Dune Paragon.

I keep meaning to open a separate bug about that...

**Comment by cooljeanius** [comment]
> > > That's worrying, AFAICS it's running a global search-and-replace for the string Irs with no context-checking.
> > 
> > 
> > It's been around far longer. Since my name keeps getting corrected to Dune Paragon.
> 
> I keep meaning to open a separate bug about that...

...ok, I opened #7881
